### PR TITLE
add new tool lcrgenie v1.0.2

### PR DESCRIPTION
**Pull request by kenza12**
Add new tool **LCRGenie v1.0.2**: https://github.com/neilswainston/LCRGenie

Thank you !!

FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/lcrgenie/lcrgenie.xml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    <stdio>
+        <regex match="WARNING:" level="warning" />
+        <regex match="ERROR:" level="fatal" />
+    </stdio>
+    <command detect_errors="exit_code"><![CDATA[
+        python -m lcr_genie
+            '$LCR_input'
+            'out.xlsx'
+            --melting_temp '$adv.melting_temp'
+        && cp 'out.xlsx' '$LCR_output'
```
Comment (bgruening) [inline tools/lcrgenie/lcrgenie.xml:19]: mv instead of cp might be more performant on a few systems.

File: tools/lcrgenie/lcrgenie.xml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    ]]></command>
+    <inputs>
+        <param name="LCR_input" type="data" format="xml" label="Constructs" help="SBOL file containing constructs designs and sequences" />
+        <section name="adv" title="Advanced Options" expanded="false">
```
Comment (bgruening) [inline tools/lcrgenie/lcrgenie.xml:23]: maybe expand by default?

File: tools/lcrgenie/lcrgenie.xml (hunk 1 above)
Comment (kenza12) [reply tools/lcrgenie/lcrgenie.xml]: Ok, done, thanks.

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks a lot @kenza12! Always awesome!

**Comment by kenza12** [comment]
> Thanks a lot @kenza12! Always awesome!

Thanks to you and all Galaxy team.

**Pull request merged**